x-pack/plugins/apm/public/components/fleet_integration/apm_enrollment_flyout_extension.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiButton, EuiText, EuiSpacer } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+
+import { AgentEnrollmentFlyoutFinalStepExtension } from '../../../../fleet/public';
+import { useKibana } from '../../../../../../src/plugins/kibana_react/public';
+import { ApmPluginStartDeps } from '../../plugin';
+
+function StepComponent() {
+  const { http } = useKibana<ApmPluginStartDeps>().services;
+  const installApmAgentLink = http?.basePath.prepend('/app/home#/tutorial/apm');
+
+  return (
+    <>
+      <EuiText>
+        <p>
+          <FormattedMessage
+            id="xpack.apm.fleetIntegration.enrollmentFlyout.installApmAgentInstructions"
+            defaultMessage="
+          After the agent starts, you can install APM agents on your hosts to collect data from your applications and services."
+          />
+        </p>
+      </EuiText>
+      <EuiSpacer size="m" />
+
+      <EuiButton fill href={installApmAgentLink}>
+        <FormattedMessage
+          id="xpack.apm.fleetIntegration.enrollmentFlyout.installApmAgentButtonText"
+          defaultMessage="Install APM Agent"
+        />
+      </EuiButton>
+    </>
+  );
+}
+
+export function getApmEnrollmentFlyoutData(): Pick<
+  AgentEnrollmentFlyoutFinalStepExtension,
+  'title' | 'Component'
+> {
+  return {
+    title: i18n.translate(
+      'xpack.apm.fleetIntegration.enrollmentFlyout.installApmAgentTitle',
+      {
+        defaultMessage: 'Install APM Agent',
+      }
+    ),
+    Component: StepComponent,
```

Comment:
@nchaulet - Here's where that refactor with title/Component really kicks in. Able to use hooks, etc, in this `StepComponent` and still place it where we need to in the Fleet UI. Great call on that.